src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaronContent.tsx

```diff
@@ -80,29 +82,26 @@ export const BuildBaronContent: React.VFC<BuildBaronCoreProps> = ({
 
   const customTickets = customCreatedTickets?.task?.annotation?.createdIssues;
   const bbTickets = bbCreatedTickets?.bbGetCreatedTickets;
+  const canCreateTickets = bbData?.bbTicketCreationDefined;
 
   return (
-    <div data-cy="bb-content">
+    <Wrapper data-cy="bb-content">
       {loading && <Skeleton active title={false} paragraph={{ rows: 4 }} />}
-      {bbData?.bbTicketCreationDefined ? (
+      {canCreateTickets ? (
         <CustomCreatedTickets
           taskId={taskId}
           execution={execution}
           tickets={customTickets}
         />
       ) : (
-        <CreatedTickets
+        <BBCreatedTickets
           taskId={taskId}
           execution={execution}
           buildBaronConfigured={bbData?.buildBaronConfigured}
           tickets={bbTickets}
         />
       )}
-      {bbTickets?.length > 0 && <BuildBaronTable jiraIssues={bbTickets} />}
 
-      {customTickets?.length > 0 && (
-        <CustomCreatedTicketsTable createdIssues={customTickets} />
-      )}
       <AnnotationNote
         note={annotation?.note}
```

Comment:
Could use some extra space in between the tickets and the annotation note!

<img width="1062" alt="Screen Shot 2022-05-04 at 8 13 13 PM" src="https://user-images.githubusercontent.com/47064971/166850204-1796007e-1420-4721-b656-7c57bff9bcfe.png">

Reply:
![image](https://user-images.githubusercontent.com/4605522/166981790-483ef5c3-e95d-4276-972f-1a7a84013073.png)
Done